imports/api/utils/NewGameUtils.js

```diff
@@ -128,7 +128,8 @@ export default {
 		                          this.someoneAboveMinimum(Constants.JPN_END_POINTS);
 		// End condition where north player reaches first place after winning on last round
 		let dealerFirstAndAboveMinimum = Session.get("current_round") == 8 &&
-		                                 Session.get("current_bonus") > 0 &&
+										 Session.get("current_bonus") > 0 &&
```

Comment:
Change alignment to spaces so it doesn't mis-align

Reply:
You still haven't fixed this alignment issue.